routes/api/brothers/brothers.controller.js

```diff
@@ -111,4 +111,28 @@ brotherController.post('/login', (req, res) => {
   });
 });
 
+/**
+ * EDIT Endpoint
+ * @route PUT api/borthers/edit
+ * @desc edit a bro page
+ */
+brotherController.put('/:brotherID', (req, res) => {
+  // Form validation
+  const { errors, isValid } = validateLoginInput(req.body);
```

Comment:
This function can't be used to validate edit form, it's for login. You'll need to write a validation function for the edit page individually, let me know if you need help w this. It's a bit tricky bc users won't edit every field most of the time, so we have to account for empty inputs accordingly